GenderPayGap.WebUI/Models/AccountCreation/CreateUserAccountViewModel.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Web.Mvc;
+using GovUkDesignSystem;
+using GovUkDesignSystem.Attributes;
+using GovUkDesignSystem.Attributes.ValidationAttributes;
+
+namespace GenderPayGap.WebUI.Models.AccountCreation
+{
+    public class CreateUserAccountViewModel : GovUkViewModel
+    {
+        [GovUkValidateRequired(
+            ErrorMessageIfMissing = "Please enter an email address."
+        )]
+        public string EmailAddress { get; set; }   
+        
+        [GovUkValidateRequired(
+            ErrorMessageIfMissing = "Please confirm your email address."
+        )]
+        public string ConfirmEmailAddress { get; set; }   
+        
+        [GovUkValidateRequired(
+            ErrorMessageIfMissing = "Please enter your first name."
+        )]
+        public string FirstName { get; set; }  
+        
+        [GovUkValidateRequired(
+            ErrorMessageIfMissing = "Please enter your last name."
+        )]
+        public string LastName { get; set; }  
+        
+        [GovUkDisplayNameForErrors(
+            NameAtStartOfSentence = "Job title",
+            NameWithinSentence = "job title"
+        )]
+        [GovUkValidateCharacterCount(
+            MaxCharacters = 50
+        )]
```

Comment:
Is there a reason you've validated the length of Job Title, but not FirstName or LastName?
They're all 50 characters in the database.
I can see a case that it's more likely that someone has a long job title than name

Reply:
I was maintaining feature parity with what currently exists on the `RegisterViewModel`. I also reasoned that it's more likely that someone has a long job title than name.

Would you like me to add a max length to FirstName and LastName?

Reply:
I think it's probably worthwhile to add the length check, even if it will rarely be used

Reply:
Done